.packaging/build_tarballs.jl

```diff
@@ -38,13 +38,14 @@ NATIVE_CMAKE_FLAGS+=(-DBC_LOAD_FLAGS="-target ${target} --sysroot=/opt/${target}
 
 cmake -B build-native -S enzyme -GNinja "${NATIVE_CMAKE_FLAGS[@]}"
 
-# Only build blasheaders (and eventually tblgen)
-ninja -C build-native -j ${nproc} blasheaders
+# Only build blasheaders and tblgen
+ninja -C build-native -j ${nproc} blasheaders enzyme-tblgen
 
 # 2. Cross-compile
 CMAKE_FLAGS=()
 CMAKE_FLAGS+=(-DENZYME_EXTERNAL_SHARED_LIB=ON)
-CMAKE_FLAGS+=(-DBC_LOAD_HEADER=`pwd`/build-native/BCLoad/gsl/blas_headers.h)
+CMAKE_FLAGS+=(-DBC_LOAD_HEADER=${WORKSPACE}/Enzyme/build-native/BCLoad/gsl/blas_headers.h)
+CMAKE_FLAGS+=(-DEnzyme_TABLEGEN_EXE=${WORKSPACE}/Enzyme/build-native/tools/enzyme-tblgen/enzyme-tblgen)
```

Comment:
```suggestion
CMAKE_FLAGS+=(-DBC_LOAD_HEADER=`pwd`/build-native/BCLoad/gsl/blas_headers.h)
CMAKE_FLAGS+=(-DEnzyme_TABLEGEN_EXE=`pwd`/build-native/tools/enzyme-tblgen/enzyme-tblgen)
```

Reply:
I think I prefer the `WORKSPACE` variant but I wanted to sync https://github.com/JuliaPackaging/Yggdrasil/pull/5175/commits/86d53331ca142a7bd3f61bb92832095aa5103c66#diff-dd855b760f8bc9dc6213752da30eae3410c4d7207f154d32cfa5992e487ecc0bR46-R47